### PR TITLE
pulse_async: reduce number of reconnect attempts

### PR DESCRIPTION
**Pull request by RobertMi21**

### CODE REVIEW

File: modules/pulse_async/pulse.c [hunk 1]
```diff
@@ -72,8 +73,8 @@ static void qh(int id, void *data, void *arg)
 	if (pa.paconn)
 		pa.paconn = mem_deref(pa.paconn);
 
-	if (paconn_start(&pa.paconn))
-		tmr_start(&pa.rc, 1000, reconnth, NULL);
+	pa.retry = 0;
+	tmr_start(&pa.rc, 1000, reconnth, NULL);
```
Comment (cspiel1) [inline modules/pulse_async/pulse.c:77]: Should'nt we try for ever, but increase the delay?
See also: `reconnth()`

Comment (RobertMi21) [reply modules/pulse_async/pulse.c]: Add delay increase between 800ms and 2000ms

Comment (cspiel1) [reply modules/pulse_async/pulse.c]: Increasing from 800ms to 2000ms has not much benefit. Tomorrow, we try a good delay and maybe we will retry for ever.

Maybe we should add a `module_event()` that informs the application that there had been `n` retries to connect to pulseaudio.

File: modules/pulse_async/pulse.c [hunk 2]
```diff
@@ -205,7 +216,9 @@ static int paconn_start(struct paconn_st **ppaconn)
  */
 static int pa_start(void)
 {
-	int err = 0;
+	int err;
+
+	pa.retry = 0;
```
Comment (cspiel1) [inline modules/pulse_async/pulse.c:222]: remove the new line

File: modules/pulse_async/pulse.c [hunk 3]
```diff
@@ -24,6 +24,7 @@
  * the pulseaudio async interface.
  */
 
+#define RECONN_DELAY 1500
```
Comment (cspiel1) [inline modules/pulse_async/pulse.c:27]: baresip uses enum instead of define for integer constants

**Pull request merged**